ui/selectors/multichain-accounts/multichain-accounts-selectors.types.ts

```diff
@@ -0,0 +1,46 @@
+import type {
+  AccountGroupId,
+  AccountWalletId,
+  AccountGroupMetadata,
+  AccountWalletMetadata,
+  AccountWallet,
+} from '@metamask/account-tree-controller';
+import { InternalAccount } from '@metamask/keyring-internal-api';
+import { AccountId } from '@metamask/accounts-controller';
+import { InternalAccountWithBalance } from '../selectors.types';
+
+export type AccountTreeState = {
+  wallets: {
+    [walletId: AccountWalletId]: AccountWallet;
+  };
+};
+
+export type InternalAccountsState = {
+  accounts: Record<AccountId, InternalAccount>;
+  selectedAccount: string;
```

Comment:
Nit: This could have been an `AccountId`, I'll update this in a next PR (if I manage to make my small suggestion working fine)